sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -313,35 +209,18 @@ windowed AS (
     AND client_id IS NOT NULL
     AND (count < 10000 OR count IS NULL)
   WINDOW
-    -- Aggregations require a framed window
     w1 AS (
       PARTITION BY
         client_id,
         submission_date,
         engine,
         source,
         type
-      ORDER BY
-        `timestamp` ASC
-      ROWS BETWEEN
-        UNBOUNDED PRECEDING
-        AND UNBOUNDED FOLLOWING
```

Comment:
I'm not sure what the purpose of this window frame.  I think this is the default behaviour unless I'm missing something

Reply:
If I'm reading correctly here, `w1` is now only used for `SUM` aggregations where order is not important. As long as that is the case, then yes defining `w1` with simply a PARTITION BY will give equivalent results.

But, if were still doing ARRAY_AGG here, order would be significant, and we'd need to specify both ORDER BY and window frame explicitly.

From https://cloud.google.com/bigquery/docs/reference/standard-sql/analytic-function-concepts#def_window_spec

> If neither the ORDER BY clause nor window frame clause are present, the window frame includes all rows in that partition.

> For aggregate analytic functions, if the ORDER BY clause is present but the window frame clause is not, the following window frame clause is used by default:

> RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW